src/core/prompts/system-prompt/__tests__/__snapshots__/anthropic_claude_sonnet_4-basic.snap

```diff
@@ -173,40 +195,44 @@ Parameters:
 - arguments: (required) A JSON object containing the tool's input parameters, following the tool's input schema
-- task_progress: (optional)  A checklist showing task progress after this tool use is completed. (See 'Updating Task Progress' section for more details)
+- task_progress: (optional) A checklist showing task progress after this tool use is completed. (See 'Updating Task Progress' section for more details)
 Usage:
 <use_mcp_tool>
 <server_name>server name here</server_name>
 <tool_name>tool name here</tool_name>
-<arguments>{
+<arguments>
+{
   "param1": "value1",
   "param2": "value2"
-}</arguments>
+}
+</arguments>
 <task_progress>Checklist here (optional)</task_progress>
 </use_mcp_tool>
 
 ## access_mcp_resource
-Description: The name of the MCP server providing the resource.
+Description: Request to access a resource provided by a connected MCP server. Resources represent data sources that can be used as context, such as files, API responses, or system information.
 Parameters:
-- server_name: (required) server name here
-- uri: (required) resource URI here
+- server_name: (required) The name of the MCP server providing the resource
+- uri: (required) The URI identifying the specific resource to access
 - task_progress: (optional) A checklist showing task progress after this tool use is completed. (See 'Updating Task Progress' section for more details)
 Usage:
 <access_mcp_resource>
-<server_name></server_name>
-<uri></uri>
+<server_name>server name here</server_name>
+<uri>resource URI here</uri>
 <task_progress>Checklist here (optional)</task_progress>
 </access_mcp_resource>
 
 ## ask_followup_question
 Description: Ask the user a question to gather additional information needed to complete the task. This tool should be used when you encounter ambiguities, need clarification, or require more details to proceed effectively. It allows for interactive problem-solving by enabling direct communication with the user. Use this tool judiciously to maintain a balance between gathering necessary information and avoiding excessive back-and-forth.
 Parameters:
 - question: (required) The question to ask the user. This should be a clear, specific question that addresses the information you need.
-- options: (optional) An array of 2-5 options for the user to choose from. Each option should be a string describing a possible answer.
+- options: (optional) An array of 2-5 options for the user to choose from. Each option should be a string describing a possible answer. You may not always need to provide options, but it may be helpful in many cases where it can save the user from having to type out a response manually. IMPORTANT: NEVER include an option to toggle to Act mode, as this would be something you need to direct the user to do manually themselves if needed.
+- task_progress: (optional) A checklist showing task progress after this tool use is completed. (See 'Updating Task Progress' section for more details)
```

Comment:
@saoudrizwan The original legacy description did not have task progress for ask_followup_question for focus chain and I believe that makes sense.